productService/swagger/products/api/swagger/swagger.yaml

```diff
@@ -0,0 +1,101 @@
+swagger: "2.0"
```

Comment:
Did you generate this file or wrote it manually? it contains errors
You can check it with https://editor.swagger.io/